### PR TITLE
feat: Detect sap/ui/core/plugin/DeclarativeSupport and sap/ui/core/plugin/LessSupport

### PR DESCRIPTION
**Pull request by d3xter666**
JIRA: CPOUI5FOUNDATION-905

### CODE REVIEW

File: src/linter/messages.ts [hunk 1]
```diff
@@ -250,21 +252,24 @@ export const MESSAGE_INFO = {
...
+	[MESSAGE.DEPRECATED_LESS_SUPPORT]: {
 		severity: LintMessageSeverity.Error,
 		ruleId: RULES["no-deprecated-api"],
 
-		message: ({libInitFunction}: {libInitFunction: string}) =>
-			`Deprecated call to ${libInitFunction}(). Use parameter {apiVersion: 2} instead`,
-		details: () => `{@link sap.ui.core.Lib.init Lib.init}`,
+		message: () => `LessSupport is deprecated.`,
+		details: () =>
+			"Not needed anymore! Use UI5 Tooling to compile Les to CSS on the fly.",
```
Comment (flovogt) [inline src/linter/messages.ts:272]:
```suggestion
			"Not required anymore! Use UI5 Tooling https://sap.github.io/ui5-tooling/stable/ to compile LESS to CSS on the fly.",
```

File: src/linter/messages.ts [hunk 2]
```diff
@@ -250,21 +252,24 @@ export const MESSAGE_INFO = {
...
+	[MESSAGE.DEPRECATED_LESS_SUPPORT]: {
 		severity: LintMessageSeverity.Error,
 		ruleId: RULES["no-deprecated-api"],
 
-		message: ({libInitFunction}: {libInitFunction: string}) =>
-			`Deprecated call to ${libInitFunction}(). Use parameter {apiVersion: 2} instead`,
-		details: () => `{@link sap.ui.core.Lib.init Lib.init}`,
+		message: () => `LessSupport is deprecated.`,
+		details: () =>
+			"Not required anymore! Use UI5 Tooling https://sap.github.io/ui5-tooling/stable/ to compile LESS to CSS on the fly.",
```
Comment (flovogt) [inline src/linter/messages.ts:272]: @KlattG. Could you please review this line

File: src/linter/messages.ts [hunk 3]
```diff
@@ -276,6 +281,23 @@ export const MESSAGE_INFO = {
...
+		message: () => `Usage of native HTML in XML Views/Fragments is deprecated`,
+		details: () => `{@link topic:be54950cae1041f59d4aa97a6bade2d8 Using Native HTML in XML Views (deprecated)}`,
+	},
+
+	[MESSAGE.LIB_INIT_API_VERSION]: {
+		severity: LintMessageSeverity.Error,
+		ruleId: RULES["no-deprecated-api"],
+
+		message: ({libInitFunction}: {libInitFunction: string}) =>
+			`Deprecated call to ${libInitFunction}(). Use parameter {apiVersion: 2} instead`,
```
Comment (flovogt) [inline src/linter/messages.ts:297]: @KlattG. Could you please review this line

File: src/linter/messages.ts [hunk 4]
```diff
@@ -276,6 +281,23 @@ export const MESSAGE_INFO = {
 		details: ({details}: {details?: string}) => details,
 	},
 
+	[MESSAGE.HTML_IN_XML]: {
+		severity: LintMessageSeverity.Error,
+		ruleId: RULES["no-deprecated-api"],
+
+		message: () => `Usage of native HTML in XML Views/Fragments is deprecated`,
+		details: () => `{@link topic:be54950cae1041f59d4aa97a6bade2d8 Using Native HTML in XML Views (deprecated)}`,
```
[the remark this answers is not in the document]
Comment (d3xter666) [reply src/linter/messages.ts:289]: This one is part of a resolved merge conflict and should have already been resolved, but it won't hurt to change it :)

[the remark this answers is not in the document]
Comment (KlattG) [reply src/linter/messages.ts:289]: Line looking good to me.

File: src/linter/messages.ts (hunk 3 above)
Comment (KlattG) [inline src/linter/messages.ts:297]:
```suggestion
			`Deprecated call to ${libInitFunction}(). Use the {apiVersion: 2} parameter instead`,
```

**Review by KlattG — CHANGES_REQUESTED** [review]
Just some minor suggestions, otherwise fine.

File: src/linter/messages.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (flovogt) [reply src/linter/messages.ts:272]: its a log message. Markdown syntax is not supported. Therefore, this log is fine.

Comment (RandomByte) [inline src/linter/messages.ts:272]: I would suggest this change to align the message with the `DEPRECATED_DECLARATIVE_SUPPORT` message

```suggestion
			"Please consider using UI5 Tooling https://sap.github.io/ui5-tooling/stable/ to compile LESS to CSS on the fly.",
```

**Review by RandomByte — COMMENTED** [review]
One suggestion but otherwise looks good to me

**Review by RandomByte — APPROVED** [review]

File: src/linter/messages.ts (hunk 3 above)
Comment (RandomByte) [reply src/linter/messages.ts]: Done in https://github.com/SAP/ui5-linter/pull/328/commits/fb76b91b6e38873bffd691d02d0c0cfe4d2bb904

**Pull request merged**